prober/http.go

```diff
@@ -260,6 +260,11 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 			Help: "Indicates if SSL was used for the final redirect",
 		})
 
+		hasOCSPGauge = prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "probe_http_ssl_has_ocsp",
```

Comment:
since it's possible to have multiple OCSP responses in a single TLS handshake, wouldn't it be better to name this `probe_http_ssl_ocsp_total`? So the value would be 0 if it's not present, instead of simply not having the metric?

Reply:
I don't know. I'm not very experienced with creating metrics. total of what? I think of this like a boolean value. 1 if there is OCSP, 0 if not. IMO it doesn't make sense to total them or add them up.

Reply:
Let me take a look at the code and figure out what it's able to report...